### DISCUSSION TITLE
June Wolfi Community Call Tomorrow!

### DISCUSSION THREAD

**Discussion by imjasonh** [wolfi-dev/os#2622] [Announcements]
:date:Wed, 6/7 @ 12pm ET
:diamonds:How to build Ruby apps quickly using melange convert
:hourglass_flowing_sand:How to save time building complex apps w/ melange files
:gem:Why building gems into apk's using melange gives ruby apps accurate SBOM results
:bouquet:Q&A!
Add invite to your calendar: https://calendar.google.com/calendar/u/0/r/eventedit/copy/aWpydnZxc2VhaHBlMzY5OWIzN2xv[…]lMjQyNzM0MDBkZTQxYTE0M2VjNjA3MDNiNDExZDc3YjFmNTM0ZWMxNWZAZw

**Comment by kirby-koo** [comment]
Looking forward to @found-it presenting!! 💯
